package.json

```diff
@@ -32,19 +32,12 @@
         "beta-chrome-mv3": "make chrome-mv3-beta browser=chrome-mv3 type=release"
     },
     "devDependencies": {
-        "@babel/cli": "^7.22.9",
-        "@babel/core": "^7.22.9",
-        "@babel/node": "^7.22.5",
-        "@babel/polyfill": "^7.10.4",
-        "@babel/preset-env": "^7.22.9",
         "@fingerprintjs/fingerprintjs": "^3.4.2",
         "@playwright/test": "^1.36.1",
         "@types/chrome": "0.0.241",
         "@types/jasmine": "^4.3.5",
         "@types/webextension-polyfill": "^0.10.1",
         "asana": "github:Asana/node-asana",
-        "babelify": "10.0.0",
-        "browserify": "^17.0.0",
```

Comment:
(Random line because GitHub doesn't let me add inline comments to unchanged lines.)

Please could you also remove the "babel" section below?